README.md

```diff
@@ -0,0 +1,16 @@
+# Satellite Ansible Roles
+
+Ansible Roles to setup / configure Red Hat Satellite. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or FAM.
```

Comment:
I'd maybe drop the `or FAM` in favor of `(FAM)` since thats more common way to indicate an acronym after defining it.